src/plone/restapi/tests/test_copymove.py

```diff
@@ -201,6 +201,7 @@ def test_move_single_object_no_auth_raises_401(self):
     def test_move_single_object_no_permission_delete_source_raises_403(self):
         api.user.grant_roles(
             username='memberuser', obj=self.folder1, roles=['Manager', ])
+        api.content.transition(obj=self.doc1, transition='publish')
```

Comment:
otherwise, the user can't see the admin's doc, and the service fails not for the 403 but because it's unable to see the source doc.